git/branch_manager.go

```diff
@@ -0,0 +1,96 @@
+package git
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/drewwyatt/git-tidy/reporting"
+)
+
+type branchDeletionError struct {
+	Branch string
+	Msg    string
+}
+
+var reporter = reporting.NewReporter()
```

Comment:
Can this be...
```suggestion
reporter := reporting.NewReporter()
```
I honestly do not know.